src/library/base/man/paste.Rd

```diff
@@ -19,23 +19,24 @@ paste0(\dots,            collapse = NULL, recycle0 = FALSE)
   \item{collapse}{an optional character string to separate the results.  Not
     \code{\link{NA_character_}}.}
   \item{recycle0}{\code{\link{logical}} indicating if zero-length
-    character arguments should lead to the zero-length
-    \code{\link{character}(0)} after the \code{sep}-phase (which turns into
-    \code{""} in the \code{collapse}-phase, i.e., when \code{collapse} is
-    not \code{NULL}).}
+    character arguments should result in the zero-length
+    \code{\link{character}(0)}. This argument interacts with \code{collapse}
+    such that under the conditions \code{recycle0} is \code{TRUE} and
+    \code{collapse} is not \code{NULL}, zero-length arguments instead output as
+    \code{""} .}
```

Comment:
A minor suggestion on spacing
```suggestion
    \code{""}.}
```

Reply:
excellent catch, thank you so much @trangdata!